WordPress/Tests/WP/DeprecatedParameterValuesUnitTest.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+// all will give an ERROR.
+
+get_bloginfo( 'home' );
+get_bloginfo( 'siteurl' );
+get_bloginfo( "text_direction" );
+echo bloginfo( 'home' );
+echo bloginfo( "siteurl" );
+echo bloginfo( 'text_direction' );
+add_settings_field( '', '', '', 'misc' );
+add_settings_field( '', '', '', 'privacy' );
+add_settings_section( '', '', '', 'misc' );
+add_settings_section( '', '', '', 'privacy' );
+register_setting( 'misc' );
+register_setting( 'privacy' );
+unregister_setting( 'misc' );
+unregister_setting( 'privacy' );
+
+// all will give a WARNING.
+$value = 'text_direction';
+get_bloginfo( $value );
+get_bloginfo( CONSTANT );
+echo bloginfo( function_name() );
+
+// Method names within a class should be fine.
```

Comment:
These can be removed. The abstract sniff is already extensively tested elsewhere.